assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -127,15 +98,17 @@ header[role="banner"] {
   background-color: gu-colour(header-dark);
   border-bottom: 1px solid gu-colour(garnett-neutral-4);
```

Comment:
don't quote me on this but I think the underline for the header should be `header-border` now